execution.bs

```diff
@@ -3303,7 +3303,7 @@ namespace std {
   class stop_token {
   public:
 <ins>    template&lt;class T>
-    using callback_type = stop_callback&lt;T>;</ins>
+      using callback_type = stop_callback&lt;T>;</ins>
```

Comment:
This extra bit of indentation seems to be extraneous.

Reply:
We usually indent by an additional level for a declaration-only template, after the template-head.

Reply:
Yes, although we're inconsistent, I agree with your "usually" after a quick scan. I withdraw the comment. (Note that I can't "resolve" since I'm neither an admin in this repo nor the PR author.)